data/saas/config/attentive_config.yml

```diff
@@ -23,15 +23,15 @@ saas_config:
     path: /v1/me
 
   endpoints:
-    - name: delete_request
+    - name: user
```

Comment:
Same as the feedback on PowerReviews, we want the endpoint name to reflect the model type that this represents, in this case `user`.